web/server/vue-cli/package.json

```diff
@@ -27,21 +27,21 @@
   },
   "dependencies": {
     "@mdi/font": "^5.9.55",
-    "codechecker-api": "file:../../api/js/codechecker-api-node/dist/codechecker-api-6.39.0.tgz",
     "chart.js": "^2.9.4",
     "chartjs-plugin-datalabels": "^0.7.0",
+    "codechecker-api": "file:../../api/js/codechecker-api-node/dist/codechecker-api-6.41.0.tgz",
```

Comment:
The package-lock.json file should contain changes only related to this new package. All the other changes should be removed. For this you can reset the changes in the package-lock.json file and rerun the `make package` target from the root folder, so it will re-generate the package-lock.json file correctly. Please do this operation after you rebased your branch.

Reply:
Yes, I have followed these steps. Maybe this now suits what is desired?

Reply:
The package-lock.json file still contains too many changes (`22,597 additions`):
![image](https://user-images.githubusercontent.com/6695818/123393110-104d9100-d59e-11eb-9549-8067b67efee6.png).
In this file I wan't to see changes similar to this (#3320):
![image](https://user-images.githubusercontent.com/6695818/123393562-7df9bd00-d59e-11eb-8789-a752444e7aef.png)

Reply:
I don't quite understand, a simple `make package` is adding all the dependencies. I have restored it to the version currently on `origin/master` but after making the packages, these changes get added inevitably. Maybe it would help if I don't add any changes from package-lock to the commit?

Reply:
Maybe the problem is that you changed the order of some dependencies in the package.json file. Try to change only the version of the codechecker-api.

Reply:
@csordasmarton These get rearrange when I run `make package`. I just tried resetting it and running `make package`. It produces the same results. So how should we proceed now? Should I not add changes from package-lock to git?

Reply:
I run the `npm install` command in the `web/server/vue-cli` directory (this is what make package target does) and it generated the following changes (and only this change):
![image](https://user-images.githubusercontent.com/6695818/123599207-a6c7bf80-d7f5-11eb-87c9-d3871d226a68.png)

My npm version: 6.14.4 (npm --version)
node version: v10.21.0 (node --version)

Maybe one of your package version is too new?
It would be good to add changes from this file to the repo so anyone who install our npm dependencies will get the same versions.

Reply:
@csordasmarton Yes I have npm version `7.11.2` and node version `v12.18.4`. So both the versions are on the latest versions. So should I downgrade the versions of the packages for the entire project? Cause I think inevitably people usually use the latest versions and these changes would be generated next time someone tries to contribute/use codechecker?

Reply:
Yes, the problem will be that in npm v7 the format of this file is changed: https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json. For now I recommend to download and use an older version of binary package so we will not update the whole file in this PR. For example I am using this version: https://nodejs.org/download/release/v10.21.0/

I also created an issue (#3372) to update this file with a newer version of npm but we need to check if everything works fine with it.